src/nni_manager/core/nniDataStore.ts

```diff
@@ -87,7 +87,19 @@ class NNIDataStore implements DataStore {
         event: TrialJobEvent, trialJobId: string, hyperParameter?: string, jobDetail?: TrialJobDetail): Promise<void> {
         this.log.debug(`storeTrialJobEvent: event: ${event}, data: ${hyperParameter}, jobDetail: ${JSON.stringify(jobDetail)}`);
 
-        return this.db.storeTrialJobEvent(event, trialJobId, hyperParameter, jobDetail).catch(
+        let timestamp: number | undefined;
+        if (event === 'WAITING' && jobDetail !== undefined) {
```

Comment:
can be simplied to if (event === 'WAITING' && jobDetail)

Reply:
Thanks, updated.